app/javascript/components/widgets/weather/sunrise-sunset.jsx

```diff
@@ -87,11 +89,11 @@ const SunriseSunset = ({ location }) => {
         timezone={timezone}
       />
       <SunriseLabel>
-        <Text>Sunrise</Text>
+        <Text>{isNight ? 'Sunset' : 'Sunrise'}</Text>
```

Comment:
Could we be cheeky and just:
```
const capitalize = R.replace(/^./, R.toUpper);
....
 <Text>{capitalize(startTime.type)}</Text>
```

or it could just be a lookup object.